client/src/app.jsx

```diff
@@ -46,29 +57,65 @@ class App extends React.Component {
   }
 
   add() {
+    console.log('ADD', this.state.totalEnrolled);
     $.ajax({
       type: 'POST',
-      url: 'http://localhost:4000/api/addTitle',
+      url: 'http://localhost:3001/api/addTitle',
       data: {total: this.state.num},
       success: () => console.log('successfully made a post')
     });
   }
 
+
   render() {
     return (
       <div>
-        <h4>Enter number of titles to populate</h4>
+        {/* <h4>Enter number of titles to populate</h4>
         <input value = {this.state.num} onChange={this.onChange} />
-        <button onClick={this.add}> Add Titles </button>
+        <button onClick={this.add}> Add Titles </button> */}
+        <div>
+          <h1 className="banner-title">
+            <Titles title={this.state.titles}/>
+          </h1>
+        </div>
+        <span className="side-bar">
+          <span className="offered">Offered By: </span>
+          <span className="university">Stanford</span>
+        </span>
         <div>
-          <Titles title={this.state.titles}/>
+          <span className="fa fa-star checked"></span>
+          <span className="fa fa-star checked"></span>
+          <span className="fa fa-star checked"></span>
+          <span className="fa fa-star checked"></span>
+          <span className="fa fa-star "></span>
         </div>
+        <div className="instructor-main">
+          <img src= {icon} className="instructor"/>
+          <span style= {{color: 'white', fontSize: '20px'}}>Andrew Ng</span>
```

Comment:
Same as above. Instructors, OfferedBys, and Testimonials are all on the Instructors Service.